src/device/get-hex-file.ts

```diff
@@ -17,19 +25,19 @@ export const getHexFileUrl = (
 ): string | undefined => {
   if (type === HexType.Bluetooth) {
     return {
-      V1: "firmware/ml-microbit-cpp-version-combined.hex",
-      V2: "firmware/MICROBIT.hex",
-      universal: "firmware/universal-hex.hex",
+      V1: hexV1,
+      V2: hexV2,
+      universal: hexUniversal,
```

Comment:
This will put hashes in file names. It's only the universal hex that gets downloaded and that uses a download attribute on the link with a "data-collection-program" name already. So I think it's fine.